types/presto-client/index.d.ts

```diff
@@ -115,6 +115,12 @@ export interface ClientOptions {
      * @default "presto"
      */
     engine?: string;
+    /**
+     * The seconds that a query is allowed to run before it starts returning
+     * results, defaults to 60 seconds. Set to null or 0 to disable.
```

Comment:
it says `null`, how it works with `number | undefined`?

Reply:
Ah, thanks for catching that. Updated the type to include null.